presto-native-execution/presto_cpp/main/expression/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+add_library(presto_expr_eval RowExpressionEvaluator.cpp)
+
+target_link_libraries(
+  presto_expr_eval
+  presto_type_converter
+  presto_types
+  presto_protocol
+  presto_http
+  velox_coverage_util
+  velox_parse_expression
+  velox_parse_parser
+  velox_presto_serializer
+  velox_serialization
+  velox_type_parser
+  ${FOLLY_WITH_DEPENDENCIES}
+  ${PROXYGEN_LIBRARIES})
```

Comment:
This library shouldn't be dependent on proxygen. We should remove this dependency.